js/ui/editors/ConstructorControlPanel.js

```diff
@@ -344,8 +400,27 @@ export class ConstructorControlPanel extends ConstructorEditorView {
     globalSettingsSetup() {
 
         this.globalSettings.appendChild(ui.em(`Configure the default behaviour of ${plugin_name}.`));
-        preferencesMenu.menu_create(this.globalSettings);
-
+        preferencesMenu.menu_create(this.globalSettings, () => {
+            if (!this.#runtime_version_dropdown)
```

Comment:
Another nitpick - please make sure to use explicit comparisons over implicit JS `!` for checking not null.